src/compile.c

```diff
@@ -336,6 +338,9 @@ static int block_bind_subblock(block binder, block body, int bindflags, int brea
     binder.first->nformals = block_count_formals(binder);
   int nrefs = 0;
   for (inst* i = body.first; i; i = i->next) {
+    if ((i->bound_flags & bindflags) == bindflags)
```

Comment:
And this is the thing that makes it go faster?  I'll review when I'm fresh.  Right now I don't get it.